scripts/extract.arrow.shapes.pl

```diff
@@ -11,7 +11,7 @@
 my(@node)      = $root -> look_down(_tag => 'table');
 my @td         = $node[1]->look_down(_tag => 'td');
 
-my @shape = map /"(.+)"/ ? $1 : (), map +($_->content_list)[0], @td;
+my @shape = grep { length $_ } map { $_->as_text() } @td;
```

Comment:
Could you switch this to the form `grep length, map $_->as_text, @td`? It's slightly more efficient in not creating block contexts, as well as more concise.